Tests.Integration.Presentation.Web/ItSystem/V2/ItSystemsApiV2Test.cs

```diff
@@ -590,6 +590,46 @@ public async Task Can_Create_Update_And_Delete_ExternalReference()
             Assert.Empty(afterDelete.ExternalReferences);
         }
 
+        [Fact]
+        public async Task Can_Get_ItSystem_Permissions()
+        {
+            //Arrange
+            var orgId = TestEnvironment.DefaultOrganizationId;
+            var token = (await HttpApi.GetTokenAsync(OrganizationRole.GlobalAdmin)).Token;
```

Comment:
using global admin as the only choice in this situation is bad as the global admin has universal access...
change to a theory with 

`GlobalAdmin, true, true, true`, `LocalAdmin, true, true, true`, `RegularUser, true, false, false`